app/models/manageiq/providers/ibm_cloud/inventory/collector/power_virtual_servers.rb

```diff
@@ -20,7 +20,20 @@ def pvm_instance(instance_id)
   end
 
   def image(image_id)
-    images_by_id[image_id] ||= images_api.pcloud_cloudinstances_images_get(cloud_instance_id, image_id)
+    begin
+      images_by_id[image_id] ||= images_api.pcloud_cloudinstances_images_get(cloud_instance_id, image_id)
+    rescue IbmCloudPower::ApiError => err
+      error_message = JSON.parse(err.response_body)["description"]
+      _log.warn("ImageID not found: #{error_message}")
```

Comment:
Unless it is actually an issue if an image is a stock image we probably don't want to be logging a warning in this case since if it is found by https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/256/files#diff-d244b92097e187e59c19aceb5f90f45f0de129bb130fa3d8d0cd1682a2e686e4R31 everything will be successful but a warning will be logged every time.

Reply:
I switched to `debug` level, since the message is already parsed out and was helpful in my dev environment.